anime-flutter/anime_galaxy/lib/anime_setting/ui/screen/anim_setting.dart

```diff
@@ -0,0 +1,233 @@
+                            fit: BoxFit.cover),
+
+                      ),
+                      child:  Container(
+                        decoration: BoxDecoration(
+                          borderRadius: BorderRadius.all(Radius.circular(3.0)),
+                          color: Colors.black.withOpacity(0.25),
+
+                        ),
+                        child: Align(
+                            alignment: Alignment.center,
+                            child: CircularImage(w: hight/8,h: hight/8,linkImg: "https://www.lamsahfannan.com/content/uploads/2017/03/3dlat.net_08_15_258a_6.jpg",)),
+
+                      ),
+                    ),
+                  )
+                ],
+              ),
+
+              SizedBox(height: 10,),
+
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                children: [
+
+
+
+                  CustomeButton(textStyle: widget.styleApp.getTextSyle(day: false),txt: "تعديل",),
+
+                  Column(
+
+
+
+                    crossAxisAlignment: CrossAxisAlignment.end,
+
+                    children: [
+
+                      Text("محمد الكلاليب",style: TextStyle(fontWeight: FontWeight.w600,color: Colors.white,fontSize: 14),),
+                      Text("مغامرات - مغامرات",style: TextStyle(color: Colors.white,fontSize: 14),)
+                    ],
+
+                  ),
+
+
+
+
+
+                ],
+              ),
+
+
+              Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  IconButton(icon:  Icon(Icons.notifications,color: Colors.white,),onPressed: (){},),
+                  Expanded(child: Text("الاشعارات",textAlign: TextAlign.start,style:  widget.styleApp.getTextSyle(day: false),)),
+
+                ],
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  IconButton(icon:  Icon(Icons.edit,color: Colors.white,),onPressed: (){},),
+                  Expanded(child: Text("الاعدادات العامة",textAlign: TextAlign.start,style: widget.styleApp.getTextSyle(day: false),)),
+
+                ],
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  IconButton(icon:  Icon(Icons.person,color: Colors.white,),onPressed: (){},),
+                  Expanded(child: Text("الحساب",textAlign: TextAlign.start,style: widget.styleApp.getTextSyle(day: false),)),
+
+                ],
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  IconButton(icon:  Icon(Icons.work,color: Colors.white,),onPressed: (){},),
+                  Expanded(child: Text("الخصوصية",textAlign: TextAlign.start,style: widget.styleApp.getTextSyle(day: false),)),
```

Comment:
Same. 
Use URL Launcher to open a link, as for now to open google, and store the link in the consts folder, maybe URLs?!! Not sure